### PR TITLE
[MERGE][FIX] mail_plugin: provide various fixes

### PR DESCRIPTION
**Pull request by std-odoo**
This merge fixes several bugs appearing when using mail plugin.
Notably

  * do not crash if customer is removed while being used in plugin;
  * fix a console error when not logging to the addin;
  * add a missing sentence to translate;
  * improve partial searches on email;

See sub commits for more details.

Task-2601837

### CODE REVIEW

File: addons/mail_plugin/static/src/to_translate/translations_outlook.xml [hunk 1]
```diff
@@ -48,4 +48,5 @@
     <string>From : %(email)s</string>
     <string>Logged from</string>
     <string>Outlook Inbox</string>
+    <string>Save the contact to create the company</string>
```
Comment (tde-banana-odoo) [inline addons/mail_plugin/static/src/to_translate/translations_outlook.xml:51]: Shouldn't it also be in pot files ?

File: addons/mail_plugin/controllers/mail_plugin.py [hunk 2]
```diff
@@ -184,11 +184,10 @@ def res_partners_search(self, search_term, limit=30, **kwargs):
         search on.
         The method returns an array containing the dicts of the matched contacts.
         """
-
         normalized_email = tools.email_normalize(search_term)
 
         if normalized_email:
-            filter_domain = [('email_normalized', '=', search_term)]
+            filter_domain = [('email_normalized', 'ilike', search_term)]
```
Comment (tde-banana-odoo) [inline addons/mail_plugin/controllers/mail_plugin.py:190]: Not sure to understand the commit message (talks about URL ?).

**Review by tde-banana-odoo — COMMENTED** [review]
Riviou !

File: addons/mail_plugin/controllers/mail_plugin.py (hunk 2 above)
Comment (std-odoo) [reply addons/mail_plugin/controllers/mail_plugin.py:190]: oops, I mean email, fixed

File: addons/mail_plugin/static/src/to_translate/translations_outlook.xml (hunk 1 above)
Comment (std-odoo) [reply addons/mail_plugin/static/src/to_translate/translations_outlook.xml:51]: good catch, fixed

**Review by std-odoo — COMMENTED** [review]
@tde-banana-odoo Changes done :)

**Comment by tde-banana-odoo** [comment]
Thanks!

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**